protoplugin/settings.gradle

```diff
@@ -0,0 +1 @@
+
```

Comment:
unnecessry file

Reply:
It's not an unnecessary file. Its expected to be ran as a separate gradle project. And a gradle project is defined by having a settings.gradle